build-test.cmd

```diff
@@ -633,7 +637,7 @@ at the install location of previous Visual Studio version. The workaround is to
 of the previous version to "%VSINSTALLDIR%" and then build.
 REM DIA SDK not included in Express editions
 echo Visual Studio Express does not include the DIA SDK. ^
-You need Visual Studio 2015 or 2017 (Community is free).
+You need Visual Studio 2015, 2017, or 2019 (Community is free).
```

Comment:
You can delete all references to VS2015 while you are on it. We dropped support for it quite a while ago.

Reply:
Should be removed now.